.circleci/config-2_1.yml

```diff
@@ -28,6 +28,12 @@ j8_small_par_executor: &j8_small_par_executor
     #exec_resource_class: xlarge
   parallelism: 1
 
+j8_small_executor: &j8_small_executor
```

Comment:
nit, why not `j8_seq_executor`?  it exists and serves the same purpose.

Reply:
It uses different resources which are needed for different tests, that's why we created the new one :-)